packages/h5p-server/src/ContentManager.ts

```diff
@@ -21,10 +22,15 @@ const log = new Logger('ContentManager');
 export default class ContentManager {
     /**
      * @param contentStorage The storage object
+     * @param contentUserDataStorage The contentUserDataStorage to delete contentUserData for content when it is deleted
      */
-    constructor(public contentStorage: IContentStorage) {
+    constructor(
+        public contentStorage: IContentStorage,
+        public contentUserDataStorage?: IContentUserDataStorage
+    ) {
         log.info('initialize');
         this.contentStorage = contentStorage;
+        this.contentUserDataStorage = contentUserDataStorage;
```

Comment:
this.contentUserDataStorage is already set as it is set by the public constructor argument